news/5021.bugfix

```diff
@@ -0,0 +1 @@
+Package name should be normalized before we use it to search if it's already installed or not
```

Comment:
This needs to be reworded to be appropriate for https://pip.pypa.io/en/latest/news/.

Reply:
Agreed. Do you have any suggestions on how to reword this? I am thinking `Canonicalize the package name before installation`.

Reply:
I have gone with `Use canonical package names while looking up already installed packages.` in https://github.com/pypa/pip/pull/8054/commits/28e09b2fe5d4fa60ec25bb6948ca6d806dd86327